### PR TITLE
Default repository

### PR DESCRIPTION
**Pull request by bakura10**
Overseeds #264 

I've added the doc, fixed one small CS and solved conflicts.

### CODE REVIEW

File: src/DoctrineORMModule/Service/ConfigurationFactory.php [hunk 1]
```diff
@@ -79,6 +79,7 @@ public function createService(ServiceLocatorInterface $serviceLocator)
             }
         }
 
+<<<<<<< HEAD
```
Comment (ins0) [inline src/DoctrineORMModule/Service/ConfigurationFactory.php]: merge conflict

File: tests/DoctrineORMModuleTest/Service/ConfigurationFactoryTest.php [hunk 2]
```diff
@@ -117,18 +116,29 @@ public function testWillNotInstantiateConfigWithInvalidNamingStrategyReference()
         $this->factory->createService($this->serviceManager);
     }
 
+<<<<<<< HEAD
```
Comment (ins0) [inline tests/DoctrineORMModuleTest/Service/ConfigurationFactoryTest.php]: merge conflict

**Comment by bakura10** [comment]
Damn. Thanks.

**Comment by danizord** [comment]
:+1:

**Pull request merged**